### DISCUSSION TITLE
Displaying lots of choices

### DISCUSSION THREAD

**Discussion by BasileAltar** [dialogic-godot/dialogic#1234] [Q&A] [answered]
Is there a way to display answer choices on multiple lines or columns ? I need to display more choices that can actually fit inside a HboxContrainer or a VBoxContainer without stretching outside the game screen :

![image](https://user-images.githubusercontent.com/50716436/191606607-57cf7dc3-50c8-493a-a3bc-35aeb8b62d2a.png)

This seems to be impossible natively, as the "theme" options only allow for vertical and horizontal layouts.

![image](https://user-images.githubusercontent.com/50716436/191601251-aed77157-276d-4bc3-8c31-6a6d9474d672.png)

Do you see a simple solution or do I have to code the grid support myself ? 

Thank you !

**Comment by Jowan-Spooner** [comment, marked as answer]
There is no easy way of doing this in 1.4, you would have to edit the DialogNode scene as well as the script, and they are quite messy.
We've been working on improving the theming/styling workflow for 2.0, so hopefully stuff like this will be easier in the future.

**Reply by BasileAltar** [reply]
Thank you :)
